### ISSUE TITLE
Factions config not saving or reloading correctly

### ISSUE THREAD

**Issue by redshirt2** [MassiveCraft/Factions#920]
Using Spigot 1.8.8, Multiverse 2.5b, Essentials Pre2.14.13, Factions 2.8, MassiveCore 2.8, permissionsEx 1.23.3, Vault 1.5.3b37 All other plugins disabled during troubleshooting.

Also, I use PerWorldPlugins 1.2.4 which keeps the Factions working on only one world.  Though for this post, it too is disabled.

Problem: after server restart, land claims aren't loaded. 

Workaround: 
before server restart, I backup the mstore folder.  
Then Restart the computer.  
In game player claims any chunk.  
on server restore mstore folder via batch file outside of server console or game.
Factions reloads the config files and all is well... until next restart.

As far as I can see, version 2.8 does not have reload command anymore. so I have no way to 'nudge' the plugin from out of game.

I wanted to know what I'm missing.  Please and thank you.

**Comment by redshirt2** [comment]
Hi guys... So I've decided to go to Bungee cord and put my factions into a single instance server.  Since then the Factions server works fine after every restart... So I'm ok, without a solution to my OP.

But, if you're thinking of tracing this issue anyway (Factions probably should work with Multiverse, I'm thinking) the problem may stem from my worlds being nested...  I have had quite a few problems with some plugins wanting WORLDS\\SkyIsle and WORLD\SkyIsle  (note the double slashes)

I was being stubborn and didn't want all my world in the root of my server directory, so I stored them in a WORLDS folder.

So If that helps...

Cheers
